adoc/chapters/programming_interface.adoc

```diff
@@ -7669,6 +7685,9 @@ When [code]#AccessTarget# is [code]#target::local#, available only when
 
 Returns a reference to the single element that is accessed by this accessor.
 
+When [code]#AccessTarget# is [code]#target::local# or [code]#target::constant#,
```

Comment:
```suggestion
When [code]#AccessTarget# is [code]#target::local# or [code]#target::constant_buffer#,
```

The target name is `constant_buffer` not `constant`.  Same for the other statements below.

Reply:
Good eye! Fixed.